### ISSUE TITLE
[CI] Failure due to gradle artifactory name

### ISSUE THREAD

**Issue by matriv** [elastic/elasticsearch#41886]
Full logs: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+artifactory/29/console

```
> Task :distribution:bwc:maintenance:buildBwcVersion
 [5.6.17] 
 [5.6.17] FAILURE: Build failed with an exception.
 [5.6.17] 
 [5.6.17] * Where:
 [5.6.17] Initialization script '/var/lib/jenkins/.gradle/init.d/init.gradle' line: 9
 [5.6.17] 
 [5.6.17] * What went wrong:
 [5.6.17] Could not find method name() for arguments [artifactory-gradle-plugins] on object of type org.gradle.plugin.repository.internal.DefaultMavenPluginRepository.
 [5.6.17] 
 [5.6.17] * Try:
 [5.6.17] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
 [5.6.17] 
 [5.6.17] * Get more help at https://help.gradle.org
 [5.6.17] 
 [5.6.17] BUILD FAILED in 13s
```
Probably related to: https://github.com/elastic/elasticsearch/pull/41857

**Comment by matriv** [comment]
One more failure failure here: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+artifactory/30/console

**Comment by jasontedor** [comment]
I can reproduce this locally, it's specific to the test builds against Artifactory *only*. I am concocting a fix.

**Comment by jasontedor** [comment]
Closed by d35dd67cfa771ac5fffe80ba9ee7932bfeedf230
